test/e2e/testdata/bad-csv/fbc_catalog.json

```diff
@@ -0,0 +1,37 @@
+{
+  "schema": "olm.package",
+  "name": "cockroach",
+  "defaultChannel": "stable"
+}
+{
+  "schema": "olm.channel",
+  "name": "stable",
+  "package": "cockroach",
+  "entries": [
+    {
+      "name": "cockroach"
+    }
+  ]
+}
+{
+  "schema": "olm.bundle",
+  "name": "cockroach",
+  "package": "cockroach",
+  "image": "quay.io/nsapse/cockroach:latest",
+  "properties": [
+    {
+      "type": "olm.gvk",
+      "value": {
+        "group": "example.com",
+        "version": "v1alpha1"
+      }
+    },
+    {
+      "type": "olm.package",
+      "value": {
+        "packageName": "cockroach",
+        "version": "1.0.0"
+      }
+    }
+  ]
+}
```

Comment:
Any idea on whether we need this FBC manifest, or can we drop it from the set of changes? I didn't see it used anywhere going through these changes.

Reply:
This is vestigial and I thought I had removed it. Thanks for catching that.